webapi/src/main/resources/application.conf

```diff
@@ -498,4 +498,13 @@ app {
         port = ${?KNORA_INSTRUMENTATION_SERVER_PORT}
         interval = 5 seconds
     }
+
+    zio-http {
```

Comment:
as zio is an implementation detail of the HTTP server I would like to rename it to `http-server`, wdyt?

Reply:
Lines 14 and 15 declares already server in this config, for akka-http. We also have akka-http-cors in line 186, so I would rather leave it as it is or rename it to zio-http-cors. Until we completely remove Akka we should make it explicit to distinguish between.

Reply:
Just because we have leaked this implementation detail into the configuration in the past this does not mean we have to continue doing this, or should we?

Reply:
we shouldn't